### DISCUSSION TITLE
Cardiotoxicity prediction via parent-metabolite relationship

### DISCUSSION THREAD

**Discussion by egmnb** [Open-Systems-Pharmacology/Forum#1418] [Questions & Problems]
Dear All,

I am a PhD student that trying to create PBPK model to predict cardiotoxicity of drugs by revealing parent-metabolite relationship.

Does anyone here ever tested the system in this regard and help me get through it and/or any example?

Warm Regards,

**Comment by StephanSchaller** [comment]
Sure, this can be done. To get you started, check out the tutorials (https://www.open-systems-pharmacology.org/#tutorials) or the esqLABS online training center: https://learn-esqlabs.talentlms.com/

**Reply by egmnb** [reply]
Thank you, Stephan. It would be hard for a newbie, but i will try my best :)

Warm Regards,
